docs/config.md

```diff
@@ -259,6 +259,113 @@ Instead of tagging, you also have the option to add metadata to instances. This
           nickname: bobbert
 ```
 
+
+## Scheduler hints
+OpenStack provides a way to define scheduler hints, which is useful to implement affinity/anti affinity. See [Nova Filter Scheduler](https://docs.openstack.org/nova/rocky/user/filter-scheduler.html) for more information.
+
+```yaml
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    generateName: openstack-node-
+    labels:
+      set: node
+  spec:
+    providerSpec:
+      value:
+        schedulerHints:
+          group: uuid of server group
+          differentHost:
+          - UUID1
+          - UUID2
+          - ...
+          sameHost:
+          - ...
+          query: |-
+            ["and",
+             [">=","$free_ram_mb","1024"],
+             [">=","$free_disk_mb","2048"]]
+```
+
+Be aware, all of the hints depend on the setup of your cloud provider. So some filters might not work in your environment.
+
+
+This example shows you how to build anti affinity for a `MachineDeployment`.
+
+First you must create a [Server Group](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/server-group.html)
+```
+$ openstack server group create sg01 --policy anti-affinity
++--------------------------------------+------+------------+---------+-------------------+---------+----------+
+| Id                                   | Name | Project Id | User Id | Policies          | Members | Metadata |
++--------------------------------------+------+------------+---------+-------------------+---------+----------+
+| 54a88567-20ae-467e-8210-3474e54ed168 | sg01 | ...        | ...     | ['anti-affinity'] | []      | {}       |
++--------------------------------------+------+------------+---------+-------------------+---------+----------+
+```
+
+You must reference this Server Group in the definition of your `MachineDeployment`
+
+```yaml
+apiVersion: cluster.k8s.io/v1alpha1
+kind: MachineDeployment
+metadata:
+  labels:
+    set: node
+  name: test
+  namespace: default
```

Comment:
Let's shorten this example a bit by only showing the relevant parts for scheduler hints

Reply:
or maybe a new file and create a link to that file..